lib/components/ActionTrigger/ActionTrigger.tsx

```diff
@@ -50,22 +58,24 @@ export const ActionTrigger: React.StatelessComponent<ActionTriggerProps> = (prop
         />;
     }
 
+    const ContainerDivWrapper = props.theme ? StyledElements.div : Attr.div;
     return (
-        <Attr.div
+        <ContainerDivWrapper
```

Comment:
Can we make this always be `StyledElements`? That way we can always manage colors at the JS level and everything else in CSS. @vincenzocaruso what do you think?

Reply:
If there no theme prop, SC will just append an class name without any styles, so we have all colors from base theme, I think we can always use styled component here.